USER-GUIDE.md

```diff
@@ -0,0 +1,484 @@
+# KCP-Edge User Guide:
+
+## Required Packages:
+   - docker
```

Comment:
Would it be accurate to replace this with
> - docker or podman linked to appear on `$PATH` as docker
?

Reply:
I have not experimented with `podman` yet  - not sure if `podman` requires any extra configuration

Reply:
We should support podman and/or other free alternatives rather than require users on a Mac to pay Docker a fee.
Any relevant thing that we do not already know, we should find out.